packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -77,34 +69,36 @@ class PinCodeBloc extends $PinCodeBloc {
             return Stream.value(_pinCode.value.length);
           },
         ).asResultStream(),
-        errorState
-            .delay(const Duration(milliseconds: 1000))
```

Comment:
What was the purpose of this delay and why was it removed?

Reply:
Essentially it was to fix another issue (PIN dots disappearing immediately on error), but it introduced the golden test issues afterwards. I've fixed both by just using placeholder values during loading/error, and clearing the internal PIN immediately on error